@xen-orchestra/web-core/lib/components/fullscreenable/FullscreenableHeader.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <header class="header">
+    <UiIcon :icon="faAngleLeft" color="info" @click="back" />
+    <slot />
+  </header>
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import { faAngleLeft } from '@fortawesome/free-solid-svg-icons'
+
+defineProps<{
+  back: () => void
+}>()
+</script>
+
+<style lang="postcss" scoped>
+.header {
+  align-items: center;
+  display: flex;
+
+  background-color: var(--background-color-primary);
+  border-bottom: 1px solid var(--color-grey-500);
+  height: 40px;
```

Comment:
For better consistency, you can use `rem` instead of `px`:
```suggestion
  border-bottom: 0.1rem solid var(--color-grey-500);
  height: 4rem;
```